render-ws-java-client/src/main/java/org/janelia/render/client/newsolver/solvers/AlternatingDomainDecompositionSolverClient.java

```diff
@@ -0,0 +1,88 @@
+package org.janelia.render.client.newsolver.solvers;
+
+import org.janelia.render.client.ClientRunner;
+import org.janelia.render.client.newsolver.DistributedAffineBlockSolver;
+import org.janelia.render.client.newsolver.setup.AffineBlockSolverSetup;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+/**
+ * A solver that alternates between solving for a given block layout and a shifted one so that
+ * all tiles are placed in the center of a block in one of the iterations.
+ * See also: additive Schwarz domain decomposition.
+ *
+ * @author Michael Innerberger
+ */
+public class AlternatingDomainDecompositionSolverClient {
+
+	private static final int N_RUNS = 4;
```

Comment:
I completely agree (same for the second remark about adding a cleanup parameter). As discussed, adding parameters to that client is a bit more involved since `DistributedAffineSolverSetup` already extends `CommandLineParameters`.
I'll address this as soon as we know that the general procedure works.

Reply:
Ahh - you are correct - [booleans have arity 0 by default](https://jcommander.org/#_boolean)

Reply:
oops - meant this comment to go above